tripy/nvtripy/frontend/ops/slice.py

```diff
@@ -95,8 +98,15 @@ def __getitem__(
         return gather(self, 0, index)
 
     index = make_list(index)
+    ellipsis_count = index.count(Ellipsis)
+    if ellipsis_count > 1:
+        raise_error("An index can only have a single ellipsis ('...')")
     if len(index) > self.rank:
         raise_error(f"Input tensor has a rank of {self.rank} but was attempted to be sliced with {len(index)} indices")
+    if ellipsis_count:
+        ellipsis_idx = index.index(Ellipsis)
+        num_slices = self.rank - len(index) + 1
+        index[ellipsis_idx : ellipsis_idx + 1] = [slice(None)] * num_slices
```

Comment:
Could we also briefly mention ellipsis in the docstring? A short code example would be good too.